umh.docs.umh.app/content/en/docs/getstarted/installation.md

```diff
@@ -56,38 +56,42 @@ please refer to our dedicated [Firewall Rules](/docs/production-guide/security/f
 3. If you are not a member, continue with sign up. Register your information and click on **SIGN UP**.
    ![Sign up page](/images/getstarted/installation/signup.png)
 
-4. Click on **+Add Instance** button.
-   ![Add Instance button](/images/getstarted/installation/dashboard.png)
+4. You should've been redirected to the **Network View**. Now click on the **+Add Instance** button at the top right corner.
 
-5. Select **Install UMH Only**.
-   ![Install UMH Only button](/images/getstarted/installation/addinstance.png)
+5. Go through the installation wizard by first clicking on **Install the UMH** and then on **UMH Classic**.
 
-6. Enter your instace name and then click on **Create my command**.
-   ![Instance name input](/images/getstarted/installation/entername.png)
+6. Configure the instance to your needs. The only required field is the **Instance Name**.
+   Optionally you can define the location of the instance according to the ISA-95 model.
+   Defining the location of the instance allows for better organization, especially when
+   managing multiple UMH instances. By assigning them to an enterprise, site, or another
+   relevant category in the ISA-95 model, you ensure proper categorization and easier
+   identification of misconfigurations.
+   For instance, if you add a UMH instance to a site named ABC but receive messages intended
+   for site ACB (likely due to a spelling mistake), the system can highlight these discrepancies.
```

Comment:
This text sounds a bit weird in my opinion. Here is my idea that is a bit clearer:

Configure your instance to meet your specific needs. The only required field is the **Instance Name**. Optionally, you can assign a location to your instance. This location fields are hierarchical and are used to organize your instances within the Management Console, providing a clearer overview. For example, you might use the physical location of the instance and add your instance to enterprice ABC. The location specification adheres to the ISA95 model.